### PR TITLE
Server update for object detection

### PR DESCRIPTION
**Pull request by brucearmstrong**
- Sample code from https://towardsdatascience.com/object-detection-and-tracking-in-pytorch-b3cf1a696a98 and https://github.com/cfotache/pytorch_objectdetecttrack.
- Uses pytorch library for object detection. Uses GPU if available, otherwise CPU only.
- Added pytorch requirements. 
- All endpoints now use common get_image_from_request function, so they now all support multiple content_type values, including those produced by "curl -F".
- Additions to server_tester.py and tcp_client.py to add up and average the server_processing_time values from the server.

### CODE REVIEW

File: FaceDetectionServer/moedx/object_detection/config/coco.data [hunk 1]
```diff
@@ -0,0 +1,6 @@
+classes= 80
+train=data/coco/trainvalno5k.txt
+valid=data/coco/5k.txt
+names=data/coco.names
+backup=backup/
+eval=coco
```
Comment (brucearmstrong) [inline FaceDetectionServer/moedx/object_detection/config/coco.data]: All code under the "config" directory is straight from https://github.com/cfotache/pytorch_objectdetecttrack.

File: FaceDetectionServer/moedx/object_detection/models.py [hunk 2]
```diff
@@ -0,0 +1,352 @@
+import torch
```
Comment (brucearmstrong) [inline FaceDetectionServer/moedx/object_detection/models.py]: Mostly straight from https://github.com/cfotache/pytorch_objectdetecttrack.

File: FaceDetectionServer/moedx/moedx/settings.py [hunk 3]
```diff
@@ -52,7 +52,7 @@
         },
     },
     'root': {
-        'level': 'INFO',
+        'level': 'DEBUG',
```
Comment (lgarner) [inline FaceDetectionServer/moedx/moedx/settings.py]: Not info?

**Comment by brucearmstrong** [comment]
@lgarner, good catch on the log level.

**Comment by brucearmstrong** [comment]
There's no license info in the code I borrowed from https://github.com/cfotache/pytorch_objectdetecttrack. Closing this PR until we figure out how to handle licensing. Will either redo based on another properly licensed project, or use something like https://git-scm.com/book/en/v2/Git-Tools-Submodules.

**Pull request closed without merging**